.github/workflows/postgres-ptr.yml

```diff
@@ -0,0 +1,71 @@
+name: Restore database from point in time to new database server
```

Comment:
I think I understand this but want to check - if/when we run this does it automatically restore over the top of the current "production" server or does this spin up a separate server?

Reply:
PTR restore spins up a separate server and restores into it.
It doesn't alter the current production server in any way.